src/Components/Components/src/CascadingParameterState.cs

```diff
@@ -28,7 +28,12 @@ public CascadingParameterState(in CascadingParameterInfo parameterInfo, ICascadi
     public static IReadOnlyList<CascadingParameterState> FindCascadingParameters(ComponentState componentState, out bool hasSingleDeliveryParameters)
     {
         var componentType = componentState.Component.GetType();
+
+        // Suppressed with "pragma warning disable" instead of WarningSuppressions.xml so ILLink Roslyn Anayzer doesn't report the warning.
```

Comment:
does "instead of WarningSuppressions.xml" still make sense?

Reply:
Nope.  But I'll let CI finish to make sure there's nothing else before I fix that.

Reply:
Fixed.